core/libs/shared/src/core_tree.rs

```diff
@@ -5,7 +5,7 @@ use hmdb::transaction::TransactionTable;
 use std::collections::HashSet;
 use uuid::Uuid;
 
-impl<F, Log> TreeLike for &mut TransactionTable<'_, Uuid, F, Log>
+impl<F, Log> TreeLike for TransactionTable<'_, Uuid, F, Log>
```

Comment:
via blanket implementations, `&TransactionTable` is `TreeLike` and `&mut TransactionTable` is `TreeLikeMut`